demos/ros_examples/scripts/fixtures/panda_arm.urdf

```diff
@@ -0,0 +1,134 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is it possible to move this urdf to the top level demos folder so its available for all the demos or does this not work because of the docker context?

Reply:
nope indeed context problem for the dockerfile